woocommerce/wc-admin
# Try using fresh-data for OrderList

Addresses #153 
This uses [`fresh-data`](https://github.com/automattic/fresh-data) for the order list data instead of `withApiData`.

To Test:

1. `npm install`
2. `npm test` and ensure the `wc-api` tests pass.
3. Load up the dashboard and click the "Orders" icon at the top-right.
4. All should be behave as before.
5. Every 5 minutes, Orders will be re-fetched and updated here.
6. Click "Fulfill" on an order with a "processing" status, it should change immediately due to an interim optimistic edit state.
